logg.go

```diff
@@ -1,9 +1,13 @@
 package sgbucket
 
-import "log"
+import (
+	"log"
+	"sync/atomic"
+)
 
 func logg(fmt string, args ...interface{}) {
-	if Logging {
+	loggingEnabled := atomic.LoadUint32(&Logging)
```

Comment:
The locking for the Logging variable should be managed within sg-bucket (i.e. SG shouldn't need to know to handle sgbucket.Logging as an atomic uint32).

I think we should add an 'SetLogging' method to sgbucket that sets this value, and does the atomic handling.